prow/plugins/releasenote/releasenote.go

```diff
@@ -205,14 +205,14 @@ func handlePR(gc githubClient, log *logrus.Entry, pr *github.PullRequestEvent) e
 	var comments []github.IssueComment
 	labelToAdd := determineReleaseNoteLabel(pr.PullRequest.Body)
 	if labelToAdd == releaseNoteLabelNeeded {
-		if !prMustFollowRelNoteProcess(gc, log, pr, prLabels, true) {
-			ensureNoRelNoteNeededLabel(gc, log, pr, prLabels)
-			return clearStaleComments(gc, log, pr, prLabels, nil)
```

Comment:
When the last param of `clearStaleComments` is nil, it will call `ListIssueComments` API to get all comments.

Reply:
Agreed, but that is the first time we list comments and we return immediately after that. We only list comments once if the conditional is true and once if the conditional is false.

Reply:
Cool! Got it. Totally agree.
/close